ir/shared/src/main/scala/org/scalajs/ir/Types.scala

```diff
@@ -197,8 +209,11 @@ object Types {
         tpe: Type, mutable: Boolean)
   }
 
-  /** No type. */
-  case object NoType extends PrimTypeWithRef
+  /** Void type, the top of type of our type system. */
+  case object VoidType extends PrimTypeWithRef
+
+  @deprecated("Use VoidType instead", since = "1.18.0")
+  lazy val NoType: VoidType.type = VoidType
```

Comment:
`NoType` is deprecated but now it refers `VoidType` but it seems that the tests are changed to use `VoidType`. Do the tests then still have coverage for `NoType`? Seems it does but not sure and thought I'd ask.

Reply:
No they don't have coverage for `NoType` but this isn't a big deal. It's mostly to help migration of whoever may directly use the IR library.

Reply:
I was wondering that too, versus just changing the code since it seemed "internal". I follow to try and learn since this project is very well run :-)